cmd/webhook/main.go

```diff
@@ -222,3 +240,15 @@ func main() {
 		newConversionController,
 	)
 }
+
+func handler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Given that all three of these funcs are the same I suggest only having one until such time as we need to provide different functionality.  What do you think?

Reply:
I was struggling deciding whether to separate them to make the code clearer or just use the same one for different paths. Yeah now I'm more agreed with yours.